test/IRGen/big_types_corner_cases.swift

```diff
@@ -335,7 +335,7 @@ public extension sr8076_QueryHandler {
     }
 
 // CHECK-LABEL-64: define{{( dllexport)?}}{{( protected)?}} swiftcc { i64, i64 } @"$s22big_types_corner_cases19sr8076_QueryHandlerPAAE13forceHandle_45query8ReturnedQyd___SbAA9BigStructVcSgtqd___tKAA0e1_F0Rd__lF"(%swift.opaque* noalias nocapture, %swift.opaque* noalias nocapture, %swift.type*{{.*}}, %swift.type*{{.*}}, i8** {{.*}}.sr8076_QueryHandler, i8** {{.*}}.sr8076_Query, %swift.opaque* noalias nocapture swiftself, %swift.error** swifterror)
-// CHECK-64: {{.*}} = call swiftcc { i64, i64 } {{.*}}(%swift.opaque* noalias nocapture {{.*}}, %swift.opaque* noalias nocapture {{.*}}, %swift.refcounted* swiftself {{.*}}, %swift.error** noalias nocapture swifterror {{.*}})
+// CHECK-64: {{.*}} = call swiftcc { i64, i64 } {{.*}}(%swift.opaque* noalias nocapture {{.*}}, %swift.opaque* noalias nocapture {{.*}}, %swift.refcounted* swiftself {{.*}}, %swift.error** noalias nocapture{{( swifterror)?}}{{( dereferenceable\(\{\{.\}\}\))?}} {{.*}})
```

Comment:
Why the addition of the check for `dereferenceable`?  Does S/390 not support the swift calling convention?

Reply:
Swift calling convention is already supported on s390x https://reviews.llvm.org/rL267823. However,`swifterror` is replaced by `dereferenceable({{.}})` on s390x.

Reply:
That sounds wrong.  The parameter should be a `swifterror` type.  `dereferenceable` would be applicable to all the targets.  This sounds like either a bug in clang or LLVM for the s390 target.

Reply:
agree, any suggestions on how to narrow down such bug?

Reply:
I would look at the output of `-emit-silgen` from the `swiftc` frontend.